README.md

```diff
@@ -11,13 +11,16 @@ subprojects that the main app depends on to provide data types, widgets, and dat
 
 ## Running
 
-Shuffleboard is installed by the WPILib Eclipse Plugins. It can be launched from the WPILib menu in Eclipse.
-It can also be run manually `java -jar C:\Users\<username\>wpilib\tools\Shuffleboard.jar`
+Shuffleboard is installed by the FRC vscode extension: [Installation Instructions]
```

Comment:
There can't be a newline or other gap between the `]` and `(` for the link to work